notebooks/initial_EDA.ipynb

```diff
@@ -0,0 +1,903 @@
+{
```

Comment:
<p>A table render would be easier with headings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>